### DISCUSSION TITLE
ESET antivirus blocks use of systembridge

### DISCUSSION THREAD

**Discussion by stiefenswiegel** [timmo001/system-bridge#3170] [General]
### Description

When installing the EXE or MSI i get a block from ESET about the software.
'A threat (Suspicious) has been found in a file that Windows Installer is trying to access'
It is about the systembridgebackend.exe.
I am unable to run/use the systembridge without my AV exploding. Is there a way to solve this without whitelisting this in ESET?

### What Platform / OS are you running?

Windows

### What version are you running?

latest

**Comment by DeFlanko** [comment]
You have to whitelist it in ESET, just like how i had to whitelist it in BitDefender. 

Both the ```systembridgebackend.exe``` and ```systembridge.exe``` -- best to do the whole folder while your at it. 

![image](https://github.com/user-attachments/assets/a51d7be8-27fa-46b8-8264-50976b2cb6db)
